source/Userdocs/Software/jLEMS.md

```diff
@@ -1 +1,42 @@
+(jlems)=
 # jLEMS
+
+Java Interpreter for the Low Entropy Model Specification language. See http://lems.github.com/LEMS for the LEMS specification.
+
+```{admonition} jLEMS is the reference implementation of LEMS
+jLEMS was developed by Robert Cannon when the LEMS language was being devised and serves at the key reference for how to implement/interpret the language.
+```
+
+## Quick start
+
+Clone this repository:
+
+    git clone git://github.com/LEMS/jLEMS.git
+
+Install with Maven:
+
+    cd jLEMS
+    mvn install
+
+Run an example:
+
+    ./lems src/test/resources/example1.xml        (Linux/Mac)
+    lems.bat src\test\resources\example1.xml      (Windows)
+
+## Source
+
+jLEMS is under development on GitHub at [https://github.com/LEMS/jLEMS](https://github.com/LEMS/jLEMS).
+
+## Documentation
+
+More information can be found [here](http://lems.github.io/LEMS/).
+
+For more details on LEMS see:
+
+Robert C. Cannon, Padraig Gleeson, Sharon Crook, Gautham Ganapathy, Boris Marin, Eugenio Piasini and R. Angus Silver,
```

Comment:
Shouldn't be here: we don't want people digging into LEMS unless they have to. If they must, they'll find this info in the other doc locations. If this has to go here, it should go in our developer docs.